metadata.rb

```diff
@@ -2,16 +2,13 @@
 maintainer       'Oregon State University'
 maintainer_email 'chef@osuosl.org'
 license          'Apache-2.0'
-chef_version     '>= 12.18' if respond_to?(:chef_version)
+chef_version     '>= 12.18'
```

Comment:
Looks like this got missed on a previous update.
```suggestion
chef_version     '>= 14'
```